xchange-core/src/test/java/org/knowm/xchange/dto/account/BalanceTest.java

```diff
@@ -2,16 +2,21 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import java.io.IOException;
 import java.math.BigDecimal;
 import org.junit.Test;
 import org.knowm.xchange.currency.Currency;
-import org.knowm.xchange.utils.ObjectMapperHelper;
 
 public class BalanceTest {
 
   @Test
   public void testSerializeDeserialize() throws IOException {
+
+    // This deliberately doesn't use FAIL_ON_UNKNOWN_PROPERTIES so that
+    // we ensure serialization/deserialization is symmetrical.
+    ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
instead of creating a new `ObjectMapper` for each test can't we just use `ObjectMapperHelper` and the `readValueStrict` method added by you?

Reply:
That's done, @walec51. Back with you.